### PR TITLE
Python automation api sample

### PR DESCRIPTION
**Pull request by AKTaylor123**

### CODE REVIEW

File: HelloWorld.python/program.py [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+clr.AddReference("System.Collections")
+clr.AddReference("System.Core")
+clr.AddReference("System.IO")
+
+if not os.path.isfile(AUTOMATION_API_DLL_PATH):
+    raise Exception(f"Couldn't find Automation API DLL at {AUTOMATION_API_DLL_PATH}.")
+
+clr.AddReference(AUTOMATION_API_DLL_PATH)
+
+if not os.path.isfile(AUTOMATION_CLIENT_DLL_PATH):
```
Comment (wei-mclarenapplied) [inline HelloWorld.python/program.py:35]: Format is different to line 28-29.

File: HelloWorld.python/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+3. Loads sets and sessions (74-85)
+4. Creates a range of variables needed (88-111)
+5. Creates a transient parameter (113-131)
+6. Creates waveform (134-143)
+7. Displays parameters in waveform (146-163)
+8. Closes clients (166-170)
+
+This program is a sample of how to control an instance Atlas that is currently running externally through the APIs available with Python.
+
+**Instructions**: Before running, start ATLAS, load a session<sup>1</sup> with parameter `vCar` in it and execute the sample application<sup>2</sup>.
```
Comment (salman-mclarenapplied) [inline HelloWorld.python/README.md]: You should add the command for people to run to install all the dependencies in the Requirements.txt file.

File: HelloWorld.python/program.py [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+connecting = True
+i = 0
+while connecting:
+    try:
+        application_service_client.Connect(
+            Path.GetFileNameWithoutExtension(AppDomain.CurrentDomain.FriendlyName)
+        )
+        connecting = False
+    except Exception:
+        print(f"Unable to connect due to {Exception}. Trying again.")
```
Comment (owentmfoo) [inline HelloWorld.python/program.py]: Need to do `except Exception as e:` in the line above and `print(f"Unable to connect due to {e}. Trying again.")` instead to display the exception that was caught.

File: HelloWorld.python/program.py [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+    if "ParameterNotAdded" in str(e):
+        print("L119 if statement worked")
+        trasient_parameter = session_service_client.GetSessionParameter(
+            session_id, "vCar2 Demo"
+        )
+        session_service_client.RemoveDataFromTransientParameter(
+            session_id, trasient_parameter.Identifier
+        )
+    else:
+        print(str(e))
```
Comment (owentmfoo) [inline HelloWorld.python/program.py]: The exception was printed already in L122?

**Review by owentmfoo — APPROVED** [review]

**Pull request merged**